Providers/Scripts/2.6x-2.7x/Scripts/nxOMSPerfCounter.py

```diff
@@ -197,14 +210,13 @@ def ReadOMSAgentConf(HeartbeatIntervalSeconds, PerfCounterObject):
 
 
 def UpdateOMSAgentConf(WorkspaceID, HeartbeatIntervalSeconds, PerfCounterObject):
-    if os.path.exists(conf_path):
-        txt = codecs.open(conf_path, 'r', 'utf8').read().encode(
-            'ascii', 'ignore')
-        LG().Log('INFO', 'Read omsagent configuration ' + conf_path + '.')
-    else:
-        LG().Log(
-            'INFO', 'No omsagent configuration file present.  Will create new  configuration file at ' + conf_path + '.')
-        txt = ''
+    """
+    Write the new values given by parameters to the OMSAgent conf file
+    """
+    txt = get_conf_path_text()
+    if txt == '':
```

Comment:
Same comment here as well. Please use string.empty instead of '' hardcoded value.

Reply:
Same comment